cypress.config.ts

```diff
@@ -135,5 +135,5 @@ export default defineConfig({
     }
   },
 
-  retries: 2
+  retries: process.env.CI ? 2 : 0
```

Comment:
This might need bringing to the attention of other devs so adding a comment - will stop Cypress from retrying locally and only retry when running in CI.

Reply:
Nice! 🎉